### ISSUE TITLE
cmdline cursor goes backwards one character when using auto completion

### ISSUE THREAD

**Issue by 0xBABA** [microsoft/terminal#3305]
# Environment

```none
Microsoft Windows [Version 10.0.18362.418]
Windows Terminal version 0.5.2762.0

Any other software?
WSL with Ubuntu 18.04.2 LTS
```

# Steps to reproduce
1. open terminal on WSL with ubuntu
2. start typing a command
3. use tab for auto-completion 
4. cursor goes back 1 place in the cmdline (visually - not really, meaning if i continue typing cmd can be executed successfully)

# Expected behavior
auto completion should not redraw the cursor

# Actual behavior
cursor seems to go back 1 place and when trying to continue typing the command it ends up in disorder. 

![bug_report](https://user-images.githubusercontent.com/22504131/67459744-f083fe00-f641-11e9-9f40-5b10972a6061.png)

**Comment by egmontkob** [comment]
It's probably due a misunderstanding between the terminal and bash on the width of the cloud symbol.

**Comment by DHowett-MSFT** [comment]
Yeah, that sounds about right. Thanks @egmontkob!
I'm wondering if this got better with 0.6? We changed how character widths are figured out. Maybe we'll agree with bash?

**Comment by 0xBABA** [comment]
I just installed version 0.6.2951.0 and it seems to work better in regards to this bug report. 
I will continue evaluating and close this in a day or two if issue seems resolved. 
Thanks @DHowett-MSFT @egmontkob

**Comment by 0xBABA** [comment]
Issue seems to be resolved with version > 0.6
